app/models/site_setting.rb

```diff
@@ -168,6 +168,55 @@ def self.Upload
     SiteSetting::Upload
   end
 
+  %i{
+    site_logo_url
+    site_logo_small_url
+    site_mobile_logo_url
+    site_favicon_url
+  }.each { |client_setting| client_settings << client_setting }
+
+  def self.site_logo_url
+    SiteSetting.logo&.url || SiteSetting.logo_url
```

Comment:
Trivial but couldn't you drop `SiteSetting` here since you're already in the context of `self`? Same for the methods below.

Reply:
Ah good point. Will update the code :+1: